packages/http-server/src/__tests__/body-params-validation.spec.ts

```diff
@@ -716,16 +731,9 @@ describe('body params validation', () => {
           type: 'https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY',
           validation: [
             {
-              location: ['body'],
-              severity: 'Error',
-              code: 'required',
-              message: "must have required property 'id'",
-            },
-            {
-              location: ['body'],
               severity: 'Error',
               code: 'required',
-              message: "must have required property 'status'",
+              message: 'Body parameter is required',
```

Comment:
It might be better to update this test to have one of the required properties (either id or status) so that the spirit of the test is still the same: throws error if there's a missing required property in the body. Otherwise this test now changes to testing the behavior if the body is missing completely.

Reply:
I believe there is a problem:
- The body used to be optional (thus change at line 689)
- There [`body required` check](https://github.com/acolombier/prism/blob/03b46ff021b84e17d5f19844a8385b2395b7d5b7/packages/http/src/validator/index.ts#L74) and the [content assertion](https://github.com/acolombier/prism/blob/03b46ff021b84e17d5f19844a8385b2395b7d5b7/packages/http/src/validator/index.ts#L110) happens at two different places

My understanding of the `describe`/`it` is that this unit test might not have been covering the part it says it does.

I have made a change which I think should reflect a bit better the usecase this test was meant to test. The [`body required` check](https://github.com/acolombier/prism/blob/03b46ff021b84e17d5f19844a8385b2395b7d5b7/packages/http/src/validator/index.ts#L74) seems to already be covered by [this test](https://github.com/acolombier/prism/blob/a8c787f893109a966b290d3ab2c8fe909fdfa5b2/packages/http-server/src/__tests__/body-params-validation.spec.ts#L569), but happy to add an other one if you think it should have thorough testing.